### PR TITLE
Feature/frontend/fix issue

### LINKED ISSUE #45: Employee page is not displaying the required fields
# Bug Report Format

---
## 1.Bug URL:

`/employee`

## 2.Date ~Created~ Discovered: 

2022.1.21

## 3.Defect Description:

The following fields/columns must appear on the Employee list/table: 
- id (numeric)
- start/hire date
- phone number (instead of email). 

## 4.Resolution: 

Add the above fields. 

## 5.Resolution Date:

Not yet resolved. 

## 6.Screenshot Bug: 

image

---

### LINKED ISSUE #47: Service display issues
# Bug Report Format

## 1.Bug URL:

`/service`

## 2.Date Created: 

2022.1.21

## 3.Defect Description:

Several small issues related to the display of services in Service module: 

- [x] display duration of service in *minutes* instead of *hours*
- [x] blocked services should appear at the bottom or be hidden from view (togglable)
- [x] services and combos should be displayed in two panels side by side

## 4.Resolution: 
See above checklist: commit hash should be appended to the list item.

## 5.Resolution Date:
Not yet resolved

## 6.Screenshot Bug: 
image

image

image

---

### PR DESCRIPTION
**Pull request by kimchhengheng**
# Description

Resolve the frontend issue of the Service and Employee

- Display combo and service separately.
- Do not pop up a dialog "Delete service would delete combo" when deleting the service 
- Remove auto populate price and duration when create combo, this would allow edit combo price and duration
- Change the Duration from Hour to Minute
- Display id phone number on Employee table

# Related Jira Issue (ID/URL)

# Related GitHub Issues

- This fixes #47.
- This fixes #45. 
- This helps #46 in the frontend.

# Checklist

- [x] Code compiles correctly
- [x] Code is covered by current tests or new tests
- [x] Code is formatted correctly
- [x] Extended the README / documentation, if necessary

---

# Comment/Discussion

### CODE REVIEW

**Review by NJLprograms — APPROVED** [review]
Sonarcloud quality passed and code is very readable! You should update the pr description to mention what this fixes.

**Comment by kevinlizh1992** [comment]
@kimchhengheng just a small comment, the employee phone number bugs out when you enter, say 514 and a dash, it autogenerates a bunch of parentheses when you try to erase the phone number, the only way is to close the window and re-add the employee

File: frontend/components/service/durationPriceItem.js [hunk 1]
```diff
@@ -45,14 +45,14 @@ const DurationPriceItem = (props) => {
                             aria-label="Always visible"
                             value={item.duration}
                             min={0}
-                            max={2}
-                            step={0.5}
+                            max={120}
```
Comment (yuxiang-zhang) [inline frontend/components/service/durationPriceItem.js:48]: The PO said maximum can go to 5-6 hours.

File: frontend/components/service/index.js [hunk 2]
```diff
@@ -10,8 +10,9 @@ import _orderBy from "lodash/orderBy";
 
 
 const ServiceComponent = (props) => {
-    const { serviceListData, toggleBlocked, deleteService, refresh, setRefresh } = props;
+    const { serviceListData, toggleBlocked, deleteService, refresh, setRefresh , comboList} = props;
```
Comment (yuxiang-zhang) [inline frontend/components/service/index.js:13]: doesn't look like the code has been formatted correctly

File: frontend/components/service/serviceCardRow.js [hunk 3]
```diff
@@ -31,18 +31,20 @@ const CardContentNoPadding = styled(CardContent)(`
...
     const handleConfirm = () => {
-        if (step === 0 && !item.hasOwnProperty('services')) {
-            setStep(1);
-        } else {
-            deleteService(item);
-            handleClose();
-        }
+        deleteService(item);
+        handleClose();
+        // if (step === 0 && !item.hasOwnProperty('services')) {
```
Comment (yuxiang-zhang) [inline frontend/components/service/serviceCardRow.js:42]: We can remove the commented code, and retrieve later via version control if needed.

**Review by kevinlizh1992 — APPROVED** [review]

**Review by yuxiang-zhang — CHANGES_REQUESTED** [review]
Changes look good, only 1 little change is needed to meet the PO requirements.

**Comment by kimchhengheng** [comment]
> @kimchhengheng just a small comment, the employee phone number bugs out when you enter, say 514 and a dash, it autogenerates a bunch of parentheses when you try to erase the phone number, the only way is to close the window and re-add the employee

fixed the issue related to that

**Comment by kimchhengheng** [comment]
resolve the conflict.

**Review by yuxiang-zhang — APPROVED** [review]

**Pull request merged**